cmd/query/app/flags.go

```diff
@@ -98,8 +99,15 @@ func AddFlags(flagSet *flag.FlagSet) {
 
 // InitFromViper initializes QueryOptions with properties from viper
 func (qOpts *QueryOptions) InitFromViper(v *viper.Viper, logger *zap.Logger) (*QueryOptions, error) {
+	defaultOptions := DefaultQueryOptions()
+	qOpts = &defaultOptions
```

Comment:
@yurishkuro is this okay to do? i'm essentially trying to start off with the default config and override whatever is set

Reply:
would it instead be better to change the callsite from 
```go
qOpts, err := new(queryApp.QueryOptions).InitFromViper(v, logger)
```
to
```go
defaultOpts := queryApp.DefaultQueryOptions()
qOpts, err := defaultOpts.InitFromViper(v, logger)
```